utils/shellcommand/hostsfile.go

```diff
@@ -25,5 +25,5 @@ func CommandSetHostAlias(hostName, ip string) string {
 }
 
 func CommandUnSetHostAlias() string {
-	return fmt.Sprintf(`sed -i "/%s/d" /etc/hosts`, DefaultSealerHostAliasAnnotation)
+	return fmt.Sprintf(`echo "$(sed "/%s/d" /etc/hosts)" > /etc/hosts`, DefaultSealerHostAliasAnnotation)
```

Comment:
The file `/etc/hosts` is mounted by docker, which means you are not allowed to change its inode from within docker container, so it will cause error `[Device or resource busy]`(see:[find-and-sed-string-in-docker-got-error-device-or-resource-busy](https://unix.stackexchange.com/questions/404189/find-and-sed-string-in-docker-got-error-device-or-resource-busy)),  when using `sealer delete`  command which will use `sed` to change `/etc/hosts`.In fact, the essence of the `sed` command is not to modify the file, but to create a new file to replace the existing file(just like directly overwrite)(see:[docker中执行sed](https://xuxinkun.github.io/2017/07/03/docker-sed-cannot-rename/)), and you can see after `sed`  the inode of the file will be changed which is not allowed in container environment with `/etc/hosts`.